app/src/main/java/com/example/snookerapi/RecyclerViewExtensions.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.snookerapi
+
+import android.graphics.Rect
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+fun RecyclerView.addHorizontalSpace(space: Int) {
```

Comment:
лучше будет использовать здесь размер из ресурсов, а не число
```suggestion
fun RecyclerView.addHorizontalSpace(@Drawable spaceRes: Int) {
```
и получать как 
`context.resources.getDimensionPixelSize(spaceRes)`